.gitlab-ci.yml

```diff
@@ -60,6 +60,23 @@ linux-stable:
     - target/release/ethstore
     - target/release/ethkey
     name: "stable-x86_64-unknown-linux-gnu_parity"
+linux-snap:
+  stage: build
+  image: ubuntu:latest
+  only:
+    - snap
+  before_script:
+    - apt install snapcraft
+  script:
+    - git submodule update --init --recursive
+    - cd scripts
+    - snapcraft
+    - sudo snap install *.snap --dangerous
```

Comment:
need for test snap package https://gitlab.ethcore.io/parity/parity/builds/53332/artifacts/download